src/components/PublicGroupList.vue

```diff
@@ -154,6 +159,33 @@ export default {
             }
             return monitor.element.sendUrl && monitor.element.url && monitor.element.url !== "https://" && !this.editMode;
         },
+
+        /**
+         * Returns formatted certificate expiry or Bad cert message
+         * @param {Object} monitor Monitor to show expiry for
+         * @returns {string}
+         */
+        formattedCertExpiryMessage(monitor) {
+            if (monitor?.element?.validCert && monitor?.element?.certExpiryDaysRemaining) {
```

Comment:
I have not checked this: 
is `monitor?.element?.certExpiryDaysRemaining` necessary in this statement?

Reply:
I'm doing this to avoid a scenario where certExpiryDaysRemaining are still loading(happens when you first turn the toggle and before saving), if not done like this it will show "Expiry  Day"

Reply:
Would `unknown` be a better option in this case?

Reply:
Yes, that sounds better to me as well. I'm creating a fix for this and there was another bug that found adding fix for that as well.